src/main/java/org/terasology/logic/players/DefaultMinimapSystem.java

```diff
@@ -78,6 +87,9 @@ public EntityRef get() {
         int seaLevel = worldGenerator == null ? 10 : worldGenerator.getWorld().getSeaLevel();
         minimapHUDElement.setHeightRange(seaLevel, seaLevel + 64);
         minimapHUDElement.setWorldProvider(worldProvider);
+        if (entityManager.getCountOfEntitiesWith(AliveCharacterComponent.class) != 0) {
```

Comment:
As `minimapHUDElement.updateAlivePlayerList` can easily deal with empty lists, I don't see why the guarding `if` expression is required here. 
In particular, I don't think that this initialize method will be called often (hopefully only once 🙄 ), so searching for all entities should be fine.